### PR TITLE
Added GitHub issue forms

### LINKED ISSUE #1823: Update Issue templates
Update existing Issue templates with GitHub Issue forms
Demo - https://github.com/github/docs/issues/new?assignees=&labels=content&template=improve-existing-docs.yaml

https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms

### PR DESCRIPTION
**Pull request by Vaibhav-Chopra-GT**
# Description

Update the `ISSUE-TEMPLATE` with GitHub issue forms.
Thanks @priyanshuone6 for the reference! Most of the content of this PR overlaps with the reference you provided.

Fixes #1823

## Type of change

Please add a line in the relevant section of [CHANGELOG.md](https://github.com/pybamm-team/PyBaMM/blob/develop/CHANGELOG.md) to document the change (include PR #) - note reverse order of PR #s. If necessary, also add to the list of breaking changes.

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Optimization (back-end change that speeds up the code)
- [ ] Bug fix (non-breaking change which fixes an issue)

# Key checklist:

- [ ] No style issues: `$ flake8`
- [ ] All tests pass: `$ python run-tests.py --unit`
- [ ] The documentation builds: `$ cd docs` and then `$ make clean; make html`

You can run all three at once, using `$ python run-tests.py --quick`.

## Further checks:

- [ ] Code is commented, particularly in hard-to-understand areas
- [ ] Tests added that prove fix is effective or that feature works

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+name: Bug-Report
```
Comment (priyanshuone6) [inline .github/ISSUE_TEMPLATE/bug_report.yml:1]:
```suggestion
name: Bug Report
```

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+name: Feature-Request
```
Comment (priyanshuone6) [inline .github/ISSUE_TEMPLATE/feature_request.yml:1]:
```suggestion
name: Feature Request
```

**Comment by Vaibhav-Chopra-GT** [comment]
@priyanshuone6 when I change it to `Bug Report` from `Bug-Report` it shows me this error
Screenshot 2022-01-28 114645

**Comment by priyanshuone6** [comment]
That is because there already exists a template with the same name in the default branch `develop`. Merging this PR will fix it.

**Review by priyanshuone6 — APPROVED** [review]

**Review by tinosulzer — APPROVED** [review]
Thanks @Vaibhav-Chopra-GT !

**Pull request merged**